io_section.tex

```diff
@@ -0,0 +1,257 @@
+evidence that scaling is preserved from the decreasing run times. The Client Ratio (CR) is very close to zero in all cases, 
+showing that model client tasks are hardly waiting at all for I/O requests to be processed and that I/O is asynchronous.
+The \% runtime penalty for I/O is low and well within the assumed machine variability. The \% memory usage penalty for I/O is approx 8-10\%. 
+
+216 and 384 node jobs crashed out within the first few timesteps with XIOS-related errors even though no I/O was
+actually happening at those timesteps. I am assuming that further tuning is needed for these higher node-count jobs.
+See the Conclusions section for suggested avenues to explore.
+
+\subsection{Set 2: In-situ Post-processing}
+
+In this set the focus is on regridding and simple time-meaning, with usage examples taken from latest Aquaplanet branch.
+The aim here is to get an idea of computational overhead in terms of runtime and memory, and general correctness rather than scaling.
+Therefore, these tests were done with a lower-resolution C96 mesh and only out to 96 nodes.
+Runs have also been done to compare run time and memory usage to the same job configuration with only regular diagnostic output and also 
+no I/O to get an idea of the additional penalty for using XIOS post-processing. 
+
+\subsubsection{Averaging}
+
+As Set 1, with the following changes: C96 mesh, 900s timestep.
+In addition to regular diagnostics, averages are output over 20 timesteps, giving 6 writes to an additional file. 
+
+\normalsize
+\begin{figure}[ht!]
+  \begin{center}
+   \includegraphics[scale=0.4]{figs/ave.eps}
+   \caption{Averaging}
+   \label{fig:fig1}
+  \end{center}
+\end{figure}
+
+\begin{table}[ht!]
+\scriptsize
+  \begin{center}
+    \caption{Averaging: Runtime}
+    \label{tab:table3}
+     \begin{tabular}{|c|c|c|c|c|c|}
+      \textbf{Nodes} & \textbf{Max CR~(\%)} & \textbf{Max Client IO~(s)} & \textbf{Runtime~(s)} & \textbf{\% incr diag IO} & \textbf{\% incr no IO}\\
+      \hline
+      24 & 0.02 & 1.63 & 382 & -11 & +11 \\ 
+      54 & 0.03 & 0.90 & 369 & +5 & +19 \\ 
+      96 & 0.04 & 2.90 & 275 & -14 & 0 \\ 
+    \end{tabular}
+  \end{center}
+\end{table}
+
+\begin{table}[ht!]
+\scriptsize
+  \begin{center}
+    \caption{Averaging: Memory Usage}
+    \label{tab:table4}
+     \begin{tabular}{|c|c|c|c|}
+      \textbf{Nodes} & \textbf{Memory Usage Gb (pp) } & \textbf{Memory Usage Gb (diag only)} & \textbf{Memory Usage Gb (no IO)} \\
+      \hline
+      24 & 82.9 & 81.8 & 77.4 \\
+      54 & 150.9 & 149.2 & 138.8 \\
+      96 & 234.1 & ? & ? \\
+    \end{tabular}
+  \end{center}
+\end{table}
+
+
+Figure \ref{fig:fig1} gives a quick overview of scaling comparing run times for jobs without I/O, with regular diagnostic output and with
+regular diagnostic output plus averaging. Scaling is still evident as run times generally decrease with the number of nodes. From
```

Comment:
Is it worth considering adding an ideal (strong, I think) scaling line to the scaling plots?